src/zeroband/train.py

```diff
@@ -422,6 +561,10 @@ def train(config: Config):
                 loss = loss / num_grad_acc_steps
 
                 inputs_ids_shape = input_ids.shape
+
+                # REMOVED: WandB logging from here - it's now done once before the grad_acc_step loop
+
+                # Now we can delete the batch data
```

Comment:
hmm I think its still better to keep at after al the grad step, I imagine that you had to move it up because of the deletete batch here right ? 

if yes lets just remove batch from the del

Reply:
this comment might be a residual artifact of my revisions and not be applicable

Reply:
feel free to delete